packages/manager/apps/cloud/client/app/dbaas/logs/detail/options/home/logs-options-home.html

```diff
@@ -4,16 +4,16 @@
         data-on-click="ctrl.goToHomePage()"
     >
     </oui-back-button>
-
-    <p class="oui-paragraph">
-        <span data-translate="log_options_intro_text"></span>
-        <span data-translate="logs_options_description_manage"></span>
-        <a
-            data-translate="logs_options_manage_link"
-            data-ui-sref="dbaas.logs.detail.options.manage"
-        ></a
-        >.
-    </p>
+    <div class="row mb-4">
```

Comment:
Using a `.row` implies that the children elements  must have a `.col-x`.
But here it is not useful.
Why not simply considering using a regular `<p>` element like:

```html
<p class="oui-paragraph">
    <span data-translate="logs_options_description_manage"></span>
    <a
        data-translate="logs_options_manage_link"
        data-ui-sref="dbaas.logs.detail.options.manage"
    ></a
    >.
</p>
```

Reply:
If it's not in this div, line is not aligned with the content following in the page. 
You will see in the source page that next content is also in a `<div class="row mb-4">`.

Also please consider that this page will disappear in a short term.

Reply:
Ok so it means that the next element is applying some margin which is responsible of some inconsistency.

We can take it into account after if necessary so it's not mandatory then.